int32 id
int32 seq
geometry_msgs/Point local_goal
bool estop